packages/qwik/src/components/Form.tsx

```diff
@@ -9,16 +9,24 @@ import type {
   FieldArrayPath,
 } from '../types';
 
+export type FormSubmitHandler<TFieldValues extends FieldValues> = PropFunction<
```

Comment:
Here I would proceed the same way as with the comment I'll add you right at the SolidJS version.

Reply:
Done, also moved it to `types/form.ts` and rename it